src/api/controllers/reportedCommentAdminController.js

```diff
@@ -0,0 +1,107 @@
+import models from '../models';
+import status from '../../helpers/constants/status.codes';
+import paginateUser from '../../helpers/generate.pagination.details';
+
+const generatePagination = paginateUser;
+
+const {
+  Comment, ReportedComment, User, Reason
+} = models;
+
+/**
+ * contains all method for an admin to read/delete/Block/unblock a comment
+ *
+ * @export
+ * @class AdminReportedCommentController
+ */
+class AdminReportedCommentController {
+  /**
+   * This function gets all the users that are stored in the User table
+   *
+   * @author Frank Mutabazi
+   * @static
+   * @param {Object} req the request
+   * @param {Object} res the response to be sent
+   * @memberof AdminReportedCommentController
+   * @returns {Object} res
+   */
+  static async getAllReportedComments(req, res) {
+    const defaultOffset = 0;
+    const defaultLimit = 10;
+    const offset = req.query.offset || defaultOffset;
+    const limit = req.query.limit || defaultLimit;
+    const result = await ReportedComment.findAll({
+      attributes: ['createdAt'],
+      include: [
+        {
+          model: Comment,
+          required: true,
+          attributes: ['body', 'articleSlug']
```

Comment:
what do you about commentId? because @Ramadhan0 is using the id to block a specific comment

Reply:
Ahhhh... That makes sense @alainburindi